client/client.go

```diff
@@ -0,0 +1,32 @@
+package client
```

Comment:
I've set it to whclient. At this point the client is not very useful, but I figure it will be important after authentication is added.